### ISSUE TITLE
please help remove anti-adblock on this site, screendrama21.icu, which cannot be blocked by adblock plus, adblock

### ISSUE THREAD

**Issue by Jokopentil** [reek/anti-adblock-killer#4581]
![20230119_122037_rmscr.jpg](https://user-images.githubusercontent.com/114223791/213365501-8892d05d-ce12-4c0f-8106-7152ace80ac8.jpg)

**Comment by Jokopentil** [comment]
Layardrama21.icu

**Comment by krystian3w** [comment]
https://github.com/reek/anti-adblock-killer/issues/4582#issuecomment-1423968880

**Comment by krystian3w** [comment]
#5480

**Comment by krystian3w** [comment]
Duplicate of https://github.com/reek/anti-adblock-killer/issues/4580
